generators/Exercises/Alphametics.cs

```diff
@@ -24,7 +25,6 @@ public Alphametics()
         protected override TestClass CreateTestClass()
         {
             var testClass = base.CreateTestClass();
-            testClass.UsingNamespaces.Add(typeof(ArgumentException).Namespace);
             testClass.UsingNamespaces.Add(typeof(Dictionary<char, int>).Namespace);
```

Comment:
Totally an opinion. At first glance, I think I would want to abstract out adding Namespaces. So in Exercise.cs maybe add another override for adding additional namespaces, and then call that during the TestClass construction. I looked at some of the other tests, and I don't think adding additional namespaces (aside from exception throwing, which you've already taken care of) is going to be incredibly common anyway.